util/tlgen/xbar_cover.cfg.tpl

```diff
@@ -49,4 +49,3 @@ begin tgl
   +tree tb.dut 1
   -node tb.dut.scanmode_i
 end
-
```

Comment:
hmm... the `fix_trailing_whitespace.py` is supposed to ignore template files, are you sure you needed this one?

Reply:
It's to fix the generated files, which it _does_ check (when they change).

Reply:
o that's right. Should we exclude `.gen.` also? :)

Reply:
If I am reading it right, we currently only exclude vendor/ directories. Everything else is checked. Hm... I don't have strong opinions about which things need to be exempted, though--I just didn't like that it was a new check with a delayed trigger, and it exploded in my face / PR, haha.